src/modules/Financial/models/scb.model.ts

```diff
@@ -43,7 +52,12 @@ const buildScbHeaders = async (includeAuth = false): Promise<HeadersInit> => {
 
     return headers;
   } catch (error) {
-    handlePrismaError(error);
+    if (error instanceof BaseError) {
+      throw error;
+    }
+    throw new InternalServerError(
+      error instanceof Error ? error.message : 'Unknown error'
+    );
```

Comment:
any reason as to why using this type of error handling? lmk if there's any bottleneck in my current implementation.